students/Steven/lesson06/mailroom04_test.py

```diff
@@ -0,0 +1,39 @@
+#! bin/user/env python3
+
+import mailroom04 as mailroom
+import os.path
+'''
+Unit testing for mailroom code
+'''
+
+def test_donor_list():
+    donor_list = mailroom.donor_list
+    assert donor_list['John Doe'] == [2000, 2]
+    assert donor_list['Jane Doe'] == [3000, 3]
```

Comment:
this typically wouldn't constitute as a test - a test should test your code and would require execute a function from your code.